thunder/__init__.py

```diff
@@ -597,11 +597,24 @@ def get_computation_and_inputs(*args, **kwargs):
                     computation_traces.append(computation_trc)
                 cs.last_computation_transformation_stop = time.perf_counter_ns()
 
+                from thunder.executors.passes import transform_for_execution as transform_for_execution_pass
+                from thunder.executors.passes import _transform_for_operator_executor_execution
+                from thunder.distributed.utils import maybe_sort_waits
+
+                with langctxs.langctx(cd.langctx):
```

Comment:
I think `langctxs.langctx(cd.langctx)` should be applied as a decorator on the `get_computation_and_inputs` function as a whole. I will create a PR for this.